components/kserve/kserve_config_handler.go

```diff
@@ -25,18 +25,22 @@ const (
 func (k *Kserve) setupKserveConfig(ctx context.Context, cli client.Client, logger logr.Logger, dscispec *dsciv1.DSCInitializationSpec) error {
 	// as long as Kserve.Serving is not 'Removed', we will setup the dependencies
 
+	var defaultDeploymentMode DefaultDeploymentMode
+	var disableIngressCreation bool
```

Comment:
The name is misleading since it states `disable` but can mean both yes and no. It should be `isDisable` or something which indicates it's a flag (only variable name in the code).

Reply:
I agree that the name is confusing but since this variable is setting the field ingress["disableIngressCreation"] that is why I named it to match.

Reply:
Then it should be something like disableIngressCreationValue